package.json

```diff
@@ -11,7 +11,7 @@
   "scripts": {
     "build": "tsc",
     "prepublishOnly": "tsc",
-    "playground": "npm -C ./playground run dev",
+    "playground": "npm run build && cd ./playground && npm install && npm run dev",
```

Comment:
Let's keep the script as-is for now. `npm install` is not suitable here because we use `yarn` for dependency management.

I'll refactor the repo into a workspace later.

```suggestion
    "playground": "npm -C ./playground run dev",
```